views/home/index.ejs

```diff
@@ -76,6 +76,11 @@
             <textarea class="form-control" ng-model="addInfo.notes"></textarea>
           </ng-tooltip>
 
+          <ng-tooltip data-label="Private Notes"
```

Comment:
Because we are calling this Private Notes, we should rename the existing notes "Public Notes", I think.